<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="myBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QListWidget" name="SourceJoints">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="TargetJoints">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SourceUp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SourceDown">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TargetUp">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TargetDown">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SourceDelete">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TargetDelete">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="SourceJointsText">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Source Joints</string>
   </property>
  </widget>
  <widget class="QLabel" name="TargetJointsText">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Target Joints</string>
   </property>
  </widget>
  <widget class="QLabel" name="StartFrameText">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Start Frame</string>
   </property>
  </widget>
  <widget class="QLabel" name="EndFrameText">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>End Frame</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="sRoot">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tRoot">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="FindSkeleton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
